changelogs/current.yaml

```diff
@@ -406,6 +406,9 @@ new_features:
   change: |
     Add the option to reduce the rate limit budget based on request/response contexts on stream done.
     See :ref:`apply_on_stream_done <envoy_v3_api_field_config.route.v3.RateLimit.apply_on_stream_done>` for more details.
+- area: tracing
+  change: |
+    Added debug log containing the number of exported spans on the OpenTelemetry tracer.
```

Comment:
Ok, I think this change log is unnecessary

Reply:
Alright, I removed it.